src/testing-policies/03-cluster-operators.md

```diff
@@ -237,34 +224,84 @@ As a matter of fact, the Pod using a custom `nginx` profile gets rejected by
 the policy too:
 
 ```console
-$ policy-testdrive --policy module.wasm --request-file pod-req-apparmor-custom.json 
+$ policy-testdrive --policy module.wasm \
+  --request-file pod-req-apparmor-custom.json
 
 Settings validation result: SettingsValidationResponse { valid: true, message: None }
 Policy evaluation results:
 ValidationResponse { uid: "", allowed: false, patch_type: None, patch: None, status: Some(ValidationResponseStatus { message: Some("These AppArmor profiles are not allowed: [\"localhost/nginx-custom\"]"), code: None }) }
 ```
 
 We can change the default behaviour and allow some chosen AppArmor to be used:
+
 ```console
 policy-testdrive --policy module.wasm \
   --request-file pod-req-apparmor-custom.json \
   --settings '{"allowed_profiles": ["runtime/default", "localhost/nginx-custom"]}'
 ```
 
 This time the request is accepted:
+
 ```console
 Settings validation result: SettingsValidationResponse { valid: true, message: None }
 Policy evaluation results:
 ValidationResponse { uid: "", allowed: true, patch_type: None, patch: None, status: None }
 ```
 
-## Wrapping up
+## Automation
+
+All these steps shown above can be automated using
+[bats](https://github.com/sstephenson/bats).
```

Comment:
Isn't this the unmaintained one? https://github.com/sstephenson/bats/issues/150 vs https://github.com/bats-core/bats-core

Reply:
Good point, I didn't know about that. I'll update also the other places where I linked this old repo